src/Fixer/ControlStructure/NoBreakCommentFixer.php

```diff
@@ -50,6 +53,19 @@ public function isCandidate(Tokens $tokens)
     /**
      * {@inheritdoc}
      */
+    protected function createConfigurationDefinition()
+    {
+        return new FixerConfigurationResolver([
```

Comment:
@keradus I'm thinking about working on a PR to write all configuration definitions this way (method chaining only) as this is now possible on 2.3. Would you accept such PR?